mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/CassandraCombinationManagerTest.java

```diff
@@ -33,11 +35,17 @@
 import org.junit.Before;
 import org.junit.BeforeClass;
 import org.junit.ClassRule;
+import org.junit.Rule;
 
 public class CassandraCombinationManagerTest extends AbstractCombinationManagerTest {
 
     @ClassRule public static DockerCassandraRule cassandraServer = new DockerCassandraRule();
 
+    @ClassRule public static DockerCassandraRestartRule cassandraRestartRule = new DockerCassandraRestartRule();
```

Comment:
please have a look at https://github.com/linagora/james-project/pull/941/files#diff-c9bcff730d848155c53692dd5c9349e8R29 : I solved that problem more elegantly by providing a static factory that allows to configure the Cassandra container lifecycle